schemas/book_label_schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE Book (
+  id int GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1) NOT NULL PRIMARY KEY,
+  genre_id int,
+  author_id int,
+  source_id int,
+  label_id int REFERENCES Label (id),
+  publish_date date,
+  archived boolean,
+  publisher varchar(50),
+  cover_state varchar(10)
+);
```

Comment:
```suggestion
CREATE TABLE Book (
  id int GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1) NOT NULL PRIMARY KEY,
  publisher VARCHAR(50),
  cover_state VARCHAR(10),
  FOREIGN KEY (id) REFERENCES item (id) ON DELETE RESTRICT ON UPDATE CASCADE
);
```

Reply:
I think this table should not contain these data. 
According to [UML Diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/images/catalog_of_my_things.png), this table should contain id, publisher, and cover_state only. 
- [ ] So, I think it will be better to add a foreign key to make a relationship between Item and book tables.